packages/remix/defaults/entry.client.react.jsx

```diff
@@ -0,0 +1,22 @@
+import { RemixBrowser } from '@remix-run/react';
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Why is this file named with `react`?

Reply:
I just used the same name as in `@remix-run/dev`. I suppose they did it like that in case they ever support a different renderer (Preact, or other).